vm/grammar.leg

```diff
@@ -130,6 +130,8 @@ Block     = 'do' SEP
           #  - body:OptStmts - '}'           { $$ = NODE2(BLOCK, body, params) }
 
 Assign    = name:ID - ASSIGN - val:Stmt     { $$ = NODE2(ASSIGN, name, val) }
+		  | name:Call - '+' - '=' - val:Stmt  { $$ = NODE2(INC, name, val) }
```

Comment:
identation here seems different vs surrounding lines, tabs vs spaces

Reply:
oh, sorry, let me fix this